cruise.umple/src/GrammarParsing.ump

```diff
@@ -19,9 +104,19 @@ class RuleBasedParser
   
   public static HashMap<String,ChoiceRule> choicerules = new HashMap<String,ChoiceRule>();
   private static HashMap<String,String> todeclare = new HashMap<String,String>();
+  
+  // List of actions for the parser to perform. This has to be static because of choicerules and todeclare,
+  // but hopefully someday this will change.
+  private static HashMap<String,ParserAction> actionedTokens = new HashMap<String,ParserAction>();
```

Comment:
I only used `HashMap` as the previous design used that as well. (choicerules and todeclare)